wrappers/javascript/shared/src/types/IndyVdr.ts

```diff
@@ -0,0 +1,131 @@
+import type {
+  AcceptanceMechanismsRequestOptions,
+  AttribRequestOptions,
+  CredentialDefinitionRequestOptions,
+  CustomRequestOptions,
+  DisableAllTransactionAuthorAgreementsRequestOptions,
+  GetAcceptanceMechanismsRequestOptions,
+  GetAttribRequestOptions,
+  GetCredentialDefinitionRequestOptions,
+  GetNymRequestOptions,
+  GetRevocationRegistryDefinitionRequestOptions,
+  GetRevocationRegistryDeltaRequestOptions,
+  GetRevocationRegistryRequestOptions,
+  GetRichSchemaObjectByIdRequestOptions,
+  GetRichSchemaObjectByMetadataRequestOptions,
+  GetSchemaRequestOptions,
+  GetTransactionAuthorAgreementRequestOptions,
+  GetTransactionRequestOptions,
+  GetValidatorInfoRequestOptions,
+  NymRequestOptions,
+  PoolCreateOptions,
+  RevocationRegistryDefinitionRequestOptions,
+  RevocationRegistryEntryRequestOptions,
+  RichSchemaRequestOptions,
+  SchemaRequestOptions,
+  TransactionAuthorAgreementRequestOptions,
+} from '../builder'
+import type {
+  PoolSubmitActionOptions,
+  PoolSubmitRequestOptions,
+  RequestSetEndorserOptions,
+  RequestSetMultiSignatureOptions,
+  RequestSetTxnAuthorAgreementAcceptanceOptions,
+} from '../indyVdr'
+import type { PrepareTxnAuthorAgreementAcceptanceOptions, RequestSetSignatureOptions } from './builderTypes'
+import type { PoolStatus, SubmitAction, SubmitRequest, Transactions, Verifiers } from './types'
+
+// TODO: proper documentation
+export interface IndyVdr {
```

Comment:
Maybe we can group the methods in this interface to make them a bit more readbable? so all `buildXXX` below each other without newline, etc...